Pipfile

```diff
@@ -20,6 +20,6 @@ softioc = {editable = true, extras = ["useful"], path = "."}
 [scripts]
 # Put coverage here so we don't interfere with debugging in the IDE
 # Put flake8 here so we don't run it during cibuildwheel tests
-tests = "python -m pytest --cov=softioc --cov-report term --flake8"
```

Comment:
The CI for this is very different, so I put the flags here so we can separate flake8 from the pytest that is running in the container. If you change this you have to change the CI to match...

Reply:
Cos we still need to run on python2 and I don't think flake8 works there...

Reply:
or some other python2 reason that I can't remember